tangy-form-item.js

```diff
@@ -623,7 +623,7 @@ export class TangyFormItem extends PolymerElement {
     let inputs = []
     this
       .querySelectorAll('[name]')
-      .forEach(input => inputs.push(input.getProps()))
+      .forEach(input => inputs.push(input.getModProps ? input.getModProps() : input.getProps()))
```

Comment:
good place to add to this ternary. to check for window.useShrinker...

Reply:
This is the spot we might check for `window['tangyFormGetModProps']` true or false to use this feature.